src/routines/level3/xgemm.cpp

```diff
@@ -33,10 +33,11 @@ Xgemm<T>::Xgemm(Queue &queue, EventPointer event, const std::string &name):
     #include "../../kernels/level3/convert_symmetric.opencl"
     #include "../../kernels/level3/convert_triangular.opencl"
     #include "../../kernels/level3/convert_hermitian.opencl"
+    }, {
+    // keep in mind C1091 in MSVC 2013
```

Comment:
Still good maybe to separate the following 2 sets of 3 includes for MSVC 2013. They contain most of the code anyway, so the issue might still be there, even if you removed some parts. Or have you tested it?